### DISCUSSION TITLE
发生未知错误，请稍后再试

### DISCUSSION THREAD

**Discussion by Danruoweifeng** [shuax/MouseInc.Settings#247] [Q&A]
![image](https://user-images.githubusercontent.com/101177263/157205752-a38dcb40-7cbe-4a86-aa4d-be04772f7329.png)

OCR识别报错，打开VPN后能够正常识别，这是什么情况？

有时候不用VPN也能正常识别，有时候就必须打开。。。。

用VPN不方便，求解

**Comment by shuax** [comment]
因为用的试用接口，可能频率有限制。
